profile/index.html

```diff
@@ -0,0 +1,330 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="Vib'n: A cool place for cool people!. Dive into a world where vibes connect, friendships form, and fun never ends. Join the community that's redefining social interaction!"
+    />
+    <title>Vib`n | Profile</title>
+    <link
+      rel="stylesheet"
+      href="node_modules/bootstrap/dist/css/bootstrap.min.css"
```

Comment:
You don't need the node_modules version if you are importing this into SASS correctly